src/Protos/wallet.proto

```diff
@@ -30,28 +31,37 @@ service WalletService {
     // Events should also be able to be emitted.. RPC or event bus?
 }
 
+message CreateReceiverDepositEndpointRequest {
+    WalletDepositEndpoint WalletDepositEndpoint = 1;
+    string reference = 2;
+}
+
+message CreateReceiverDepositEndpointResponse {
+    project_origin.common.v1.Uuid ReceiverId = 1;
```

Comment:
Has there been considerations about creating a message just for the ReceiverId? A bit similar to the FederatedStreamId. This will make the ReceiverId strongly typed and clients will (hopefully) make fewer mistakes.

Reply:
@MartinSchmidt what do you think?